src-electron/zcl/zcl-loader-silabs.js

```diff
@@ -486,6 +490,22 @@ function prepareCluster(cluster, context, isExtension = false) {
           }
         })
       }
+      if (ev.isFabricSensitive) {
+        if (!ev.fields) {
```

Comment:
Same comment as above: i don't mind Zap being "smart" to automatically add stuff based on some logic, but it needs to be data driven from the SDK.